scripts/cromwell/get_output_paths.py

```diff
@@ -0,0 +1,326 @@
+#!/bin/python
+
+import argparse
+from os.path import isfile, getsize
+import logging
+import json
+from google.cloud import storage
+import re
+
+
+"""
+Summary: find GCS paths for specified outputs for multiple batches without downloading metadata
+
+Usage:
+  python get_output_paths.py [-w workflows.tsv | -i workflow-id] -f filenames.json -o output.tsv 
+            -b gs://bucket/workflow-name [-l LEVEL] [-s] [-k] [-e entities.txt]
+
+Parameters:
+  -w WORKFLOWS, --workflows WORKFLOWS
+                        TSV file (no header) with batch (or sample) names and
+                        workflow IDs (one workflow per batch). Either -i or -w
+                        required. -i takes precedence if both provided.
+  -i ID, --id ID        Workflow ID (alternative to -w if only one workflow).
+                        Either -i or -w required. -i takes precedence if both
+                        provided.
+  -f FILENAMES, --filenames FILENAMES
+                        JSON file with output names and filename suffixes
+                        (assumes ONE file per output, not an array)
```

Comment:
I added some additional help text to try to make this clearer, but I am hesitant to call it outputs-json because I feel like that's confusing since there's already an output-file argument. Let me know if what I've done is enough